src/Scouting.elm

```diff
@@ -30,17 +30,23 @@ type alias Model =
     , team : Maybe Int
     , match : Maybe Int
     , driverStation : String
-    , isStarted : Bool
+    , isStarted : ButtonState
     }
 
 
+type ButtonState
+    = Pushed
+    | TriedPush
+    | Untouched

```

Comment:
Find better names.
Just like you talk about FRC "teams" and "robots" and not about "groups of students" and "(usually) remote-controlled machines" - use terms from the "busyness-logic" domain.